lib/valid_field.ex

```diff
@@ -27,6 +27,38 @@ defmodule ValidField do
   end
 
   @doc """
+  Will assert if the given field's current value in the changeset is valid or not.
+
+  ## Examples
+    iex> ValidField.with_changeset(%Model{first_name: "Test"})
+    ...> |> ValidField.assert_valid_field(:first_name)
+    iex> ValidField.with_changeset(%Model{})
+    ...> |> ValidField.assert_valid_field(:first_name)
+    ** (ExUnit.AssertionError) Expected the following values to be valid for "first_name": nil
```

Comment:
I suspect the rest of the library will have to be updated as well. I just copy/pasted from the other methods